src/store/slices/login.slice.ts

```diff
@@ -0,0 +1,44 @@
+import { Action } from "../../models";
+const LOGIN_USER = "LOGIN_USER";
+const LOGOUT_USER = "LOGOUT_USER";
+
+interface authState {
+  user: {
+    isAuth: boolean;
```

Comment:
dont we need to store user id also in the user object?

Reply:
For now, since we dont need the user id, I didn't provide it.